youslist.txt

```diff
@@ -1928,6 +1931,7 @@
 ||ph.spotvnews.co.kr/adcontent/script/*.js^$domain=m.spotvnews.co.kr|spotvnews.co.kr
 ||photo.coolenjoy.net/banner/$domain=coolenjoy.net|cooln.kr|on.coolenjoy.net
 ||physicsworld.com/wp-content/themes/iopp/assets/js/realmedia.js^$domain=physicsworld.com
+||pics.auction.co.kr/common/blank.gif^$domain=auction.co.kr
```

Comment:
https://github.com/AdguardTeam/AdguardFilters/issues/176488 ( https://github.com/yous/YousList/commit/a14f3e21276a7ba7ff2b9baa72952ff7373bc1ca )
The filter causes the above issue.

Reply:
Yes, a14f3e2 was added to fix the issue. Please remove this line.

```suggestion
```